### ISSUE TITLE
statistics not defined

### ISSUE THREAD

**Issue by tomasenskede** [nordicopen/easee_hass#205]
### Question

HA 2022.8.7, trying to add sensor.hermanssons_equalizer_import_energy to the Energy (:8123/config/energy) dashboard... gets error;

statistics_not_defined
sensor.hermanssons_equalizer_import_energy

Entity unavailable
The state of these configured entities are currently not available:
sensor.hermanssons_equalizer_import_energy (unavailable)

### What version of the integration are you using?

v0.9.44

**Comment by olalid** [comment]
I think you just need to wait for a while until there is enough data collected.
The energy sensors only updates once per hour normally (limitation of the standard used in the electric meters) so you need to have a few hours pass.
Let us know if does not work after 24 hours.

**Comment by tomasenskede** [comment]
Three days later, I still have the same error

statistics_not_defined

  *   sensor.hermanssons_equalizer_import_energy

Entiteten är inte tillgänglig
Tillståndet för dessa konfigurerade entiteter är för närvarande inte tillgängligt:

  *   sensor.hermanssons_equalizer_import_energy (unavailable)

I have data on the sensor in all other dashboards... only the "Energy" dashboard that doesn't have it...

Tomas

________________________________
Från: Ola Lidholm ***@***.***
Skickat: den 26 augusti 2022 11:38
Till: fondberg/easee_hass
Kopia: Tomas Hermansson; Author
Ämne: Re: [fondberg/easee_hass] statistics not defined (Issue #205)


I think you just need to wait for a while until there is enough data collected.
The energy sensors only updates once per hour normally (limitation of the standard used in the electric meters) so you need to have a few hours pass.
Let us know if does not work after 24 hours.

**Comment by olalid** [comment]
I.e. history like this:
<img width="418" alt="Screenshot 2022-08-29 at 13 53 50" src="https://user-images.githubusercontent.com/34105674/187195307-0796d31d-3cdd-4c6c-af33-16d4f31472ff.png">

**Comment by tomasenskede** [comment]
The history is there:
***@***.***

But the dashboard has no information at all
***@***.***

These are my settings;

***@***.***


Från: Ola Lidholm ***@***.***>
Skickat: den 29 augusti 2022 13:54
Till: fondberg/easee_hass ***@***.***>
Kopia: Tomas Hermansson ***@***.***>; Author ***@***.***>
Ämne: Re: [fondberg/easee_hass] statistics not defined (Issue #205)


I.e. history like this:
[Screenshot 2022-08-29 at 13 53 50]<https://user-images.githubusercontent.com/34105674/187195307-0796d31d-3cdd-4c6c-af33-16d4f31472ff.png>

**Comment by tomasenskede** [comment]
Ops… is this the root cause of the error maybe? “energy” dosnt have data, “power” has… if so, then the next question… why isn’t “energy” updated? Easee plugin issue`?

***@***.***

Från: Ola Lidholm ***@***.***>
Skickat: den 29 augusti 2022 13:54
Till: fondberg/easee_hass ***@***.***>
Kopia: Tomas Hermansson ***@***.***>; Author ***@***.***>
Ämne: Re: [fondberg/easee_hass] statistics not defined (Issue #205)


I.e. history like this:
[Screenshot 2022-08-29 at 13 53 50]<https://user-images.githubusercontent.com/34105674/187195307-0796d31d-3cdd-4c6c-af33-16d4f31472ff.png>

**Comment by olalid** [comment]
I guess you tried to attach some pictures by replying to the email? Seems like it did not work which makes it very hard to follow...

**Comment by niclasdahlquist** [comment]
"Ops… is this the root cause of the error maybe? “energy” dosnt have data,", 
I think you are right on this one. I have the same issue and it is only the "energy" sensors that are available from the Energy card, not "power"
I get this errror when setting this. I hope the attchment is visible.

Br
//Niclas

![Energy_error](https://user-images.githubusercontent.com/63284661/187269191-2a7d820b-dfea-45d0-8f22-18d27bce9cb2.JPG)

**Comment by niclasdahlquist** [comment]
This is the view from Integration, where you can also see "unavailable" and below category "Diagnostik"??!
The Power sensors seem to work fine, but as I said, these are not available in drop down when configuring Energy card, only the "Energy", which are unavailable.  
![Energy_error_2](https://user-images.githubusercontent.com/63284661/187271602-b46e462d-6482-4e97-9ba1-5c98a17b0eee.JPG)

**Comment by olalid** [comment]
I assume your sensors are not disabled, right?
<img width="436" alt="Screenshot 2022-08-29 at 22 23 06" src="https://user-images.githubusercontent.com/34105674/187291622-2913d96f-4f57-442b-a374-f3f3fc62aa5f.png">

Unless that is the case I think this is likely a cloud and/or issue with compatibility between equalizer and electric meter.
Also, please check if you can see these values in the easee web interface. If you can not see them there either, the problem is most likely in the Easee cloud.

**Comment by niclasdahlquist** [comment]
yes, they are activated.
Not exactly sure what value to correspond to from easee web, is it "Kumulativ aktiv strömimport/export". The are right now 0 so it seems like a compatibility issue as you say. It is a completely new meater from Eon (E360). So, only option is to report to Easee instead then or any other suggestions of workaround?
![easee_web](https://user-images.githubusercontent.com/63284661/187297343-b86986da-e871-4b70-85a5-73a7b14fb2e1.JPG)

**Comment by olalid** [comment]
Yes, it is "Kumulativ aktiv strömimport/export" that corresponds to these sensors.
(I do not really like their translations, makes no sense to me, it is energy, not current...).
The only thing we can do is report to Easee. Their support is usually good, but if you feel you get nowhere let me know.

**Comment by niclasdahlquist** [comment]
Good news. Got a quick respons from Easee support that they were aware of the issue and a fix is included in next FW upgrade. Do not know of their release schedule but hopefully soon :-)

**Comment by olalid** [comment]
Good, thanks for letting us know.

**Comment by tomasenskede** [comment]
Thanks, will the new firmware release be updated automatically or should check somewhere?

Tomas Hermansson
076-7218145

Sent from my mobile please excuse brevity and typos

> 31 aug. 2022 kl. 19:31 skrev niclasdahlquist ***@***.***>:
> 
> Good news. Got a quick respons from Easee support that they were aware of the issue and a fix is included in next FW upgrade. Do not know of their release schedule but hopefully soon :-)

**Comment by olalid** [comment]
It is usually updated automatically.

**Comment by niclasdahlquist** [comment]
Not sure if it is released yet, but I pushed for a release date and then
they actually pushed out the (test) new version to my specific device (very
service minded 😊), version v111 and can just confirm that everything now
works fine in HA with both import export data in HA. So, if you cannot
wait, maybe you could ask for the same.





tis 11 okt. 2022 kl. 20:02 skrev Ola Lidholm ***@***.***>:
